File: Python_Programming/Day_5/program24.py
'''
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4
    Output : * * * *
'''
def Display(No):

    for i in range(1,No+1 , 1):
        print("*\t" , end = "")


def main():
    Value = 0

    print("Enter the frequency : ",end = "")
    Value = int(input())

    Display(Value)

if __name__ == "__main__":
    main()

File: CPP_Programming/Day_5/program24.cpp
/*
    Description : Write a program that accprts frequency & prints
                  the below pattern : 
    Input : 4
    Output : * * * *
*/

#include<iostream>
using namespace std;

class Pattern
{
    public :
        int iNo;

    Pattern(int iNo)
    {
        this->iNo = iNo;
    }

    void Display()
    {
        int iCnt = 0;
        
        for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
        {
            cout<<"*\t";
        }

    }

};

int main()
{
    int iValue = 0;

    cout<<"Enter the frequency : ";
    cin>>iValue;

    Pattern pobj(iValue);

    pobj.Display();


    return 0;
}